docs/spark-procedures.md

```diff
@@ -791,6 +792,22 @@ second snapshot deleted 1 record.
 |2	| Bob	   |INSERT	|0	|5390529835796506035|
 |1	| Alice  |DELETE	|1	|8764748981452218370|
 
+Create a changelog view that computes net changes. It removes intermediate changes and only outputs the net changes. 
+```sql
+CALL spark_catalog.system.create_changelog_view(
+  table => 'db.tbl',
+  options => map('end-snapshot-id', '87647489814522183702'),
+  net_changes => true
+)
+```
+
+With the net changes, the above changelog view only contains the following row since Alice was inserted in the first snapshot and deleted in the second snapshot.
+
+|  id	| name	  |_change_type |	_change_ordinal	| _change_snapshot_id |
+|---|--------|---|---|---|
+|2	| Bob	   |INSERT	|0	|5390529835796506035|
+
+
```

Comment:
Can't add comment on exact line, but do we want to remove the reference to remove_carry in below section (Carry-over rows) and replace with what is the new recommendation?

Reply:
Removed the option, and added an example for changelog table.